src/textual/color.py

```diff
@@ -158,6 +160,10 @@ def from_rich_color(cls, rich_color: RichColor) -> Color:
             Color: A new Color.
         """
         r, g, b = rich_color.get_truecolor()
+        if rich_color.type is ColorType.STANDARD:
+            return cls(r, g, b, math.nan)
+        elif rich_color.type is ColorType.DEFAULT:
+            return cls(1, 1, 1, math.nan)
```

Comment:
(I think this might be the controversial change from this PR, as a lot of rich color names are made into ansi colors)